doc/sphinx/user/methods/nullspace-removal.md

```diff
@@ -19,27 +19,45 @@ ASPECT allows the user to specify if their model has a nullspace.
 If so, any power in the nullspace is calculated and removed from the solution after every timestep.
 There are two varieties of nullspace removal implemented: removing net linear/angular momentum, and removing net translations/rotations.
 
-For removing linear momentum we search for a constant velocity vector $\bf c$ such that $$\int_\Omega \rho ({\bf u - c}) = 0$$
+For removing linear momentum we search for a constant velocity vector $\bf c$ such that
```

Comment:
This one reads better in the new form. In general the aim is to make the equation readable, both in its source form (in the md file) and when rendered on the website.